homeassistant/components/cover/mqtt.py

```diff
@@ -58,6 +81,8 @@ def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
         config.get(CONF_NAME),
         config.get(CONF_STATE_TOPIC),
         config.get(CONF_COMMAND_TOPIC),
+        config.get(CONF_TILT_COMMAND_TOPIC),
+        config.get(CONF_TILT_STATUS_TOPIC),
```

Comment:
Do you mean to make a schema for the MQTT cover or add it to the base RW one? I might need help with this.